packages/dev/core/src/Meshes/Builders/textBuilder.ts

```diff
@@ -183,7 +183,15 @@ function CreateShapePath(
     return { offsetX: glyph.ha * scale, shapePath: shapePath };
 }
 
-function CreateShapePaths(text: string, size: number, resolution: number, fontData: IFontData) {
+/**
+ * Creates shape paths from a text and font
+ * @param text the text
+ * @param size size of the font
+ * @param resolution resolution of the font
+ * @param fontData defines the font data (can be generated with http://gero3.github.io/facetype.js/)
+ * @returns array of ShapePath objects
+ */
+export function CreateTextShapePaths(text: string, size: number, resolution: number, fontData: IFontData) {

```

Comment:
The function should not be renamed to avoid breaking changes.

Reply:
Yes I know, but this function was private and was used only once in the same source file ;-)

Reply:
Ah yes, I did not see the function was not exported previously!

All good then.